tests/unit/test_logging.py

```diff
@@ -46,3 +47,37 @@ def test_verbosity_level(
     assert (
         not unexpected_log_levels
     ), f"Unexpected log level(s): {unexpected_log_levels}"
+
+
+@pytest.mark.parametrize(
+    "verbosity_level,is_progress_bar_shown",
+    [("0", False), ("1", True)],
+)
+def test_no_progress_graphic_with_min_verbosity(
+    runner,
+    test_data_upload_path,
+    tmp_path,
+    verbosity_level,
+    is_progress_bar_shown,
+):
+    """Test that progress bar is not shown when verbosity is set to 0 (errors only)."""
```

Comment:
```suggestion
    """Test that on a successful run, no output is shown when verbosity is set to 0."""
```
Wouldn't make this about progress bar